ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -226,60 +152,91 @@ The list displays all workflow versions of the selected workflow. Depending on t
 The workflow diagram includes the following elements:  
 
 - Place    
-    Intermediate stage in the workflow, represented by a circle, usually provided with incoming and/or outcoming arcs. It serves as link between two transitions.   
-    If you click a place in the workflow editor, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+    Intermediate stage in the workflow, represented by a circle, usually provided with incoming and/or outcoming arcs. It serves as link between two transitions. For detailed information, see [Places](../Overview/04_WorkflowProcessElements.md#places).   
+    If you click a place in the workflow editor, it becomes highlighted and the corresponding element settings are displayed in the settings sidebar.
 
-    ![Settings side bar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar01.png "[Settings side bar]")
+    ![Settings sidebar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar01.png "[Settings sidebar]")
 
     - *Key*  
-        Place key. This field is read-only. The key is required for API access and must be unique within the workflow version.
+        Place key. This field is read-only. The key is required for API access and must be unique within the workflow version. <!--- Stimmt das, dass er unique sein muss? Beim Import Channels order..ist der Key nicht unique.. p-out-0, p-id-0p-businessDocument-0...-->
 
```

Comment:
Da bin ich überfragt, also besser Julian/Oli fragen. Trotzdem habe ich Import Channels order kurz geprüft, und kein Place key hat sich wiederholt (in der Workflow Version). Ich vermute, das ist was gemeint ist. Oder was meinst du?